example/gpt2/huggingface_impl/gpt2_hug_impl.py

```diff
@@ -0,0 +1,272 @@
+import datetime
```

Comment:
we don't use this file within MLgym right?

Reply:
Will remove the whole folder of huggingface_impl in the gpt2 example. That was only done for paper. No need in master :)